resources/graphics.py

```diff
@@ -161,6 +161,11 @@
 				'compareModeSideBySide',
 				'wipeDisabled',
 				'wipeEnabled',
+				'catalogueOutputHeader',
+				'nodeSetCatalogueOutput1',
+				'nodeSetCatalogueOutput2',
+				'nodeSetCatalogueOutput3',
+				'nodeSetCatalogueOutput4',
```

Comment:
These are really soft on my monitor, especially the header one. When making icons this small, it makes a huge difference whether or not the objects are aligned to the pixel grid. In this case the border is 1.5 pixels, so naturally soft, and the # mark is completely unaligned. I experimented with adjusting to the grid in https://github.com/johnhaddon/gaffer/commit/04b44988d579bffd3ec8bc735500c777f4339fbc - still not great, but a bit better I think? I wonder if we'd be better off using a square than a circle?

Also, the icons are in the ImageView section - they'd be better off in the Catalogue section.

Reply:
Oh, and we should drop the `nodeSet` prefix.